proto/agent.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package agent;
+option go_package = "/internal/agent/generated/protoagent";
+
+message MetricRequest {
+    string type = 1;
+    string name = 2;
+    string value = 3;
+}
+
+message MetricResponse {
+    string status = 1;
+}
+message MetricJSONRequest {
+    string json_string = 1;
```

Comment:
Please add `buf lint` in CI
https://github.com/bufbuild/buf-lint-action

Reply:
buf-lint-action deprecated added buf-action